scalactic-macro/src/main/scala/org/scalactic/BooleanMacro.scala

```diff
@@ -802,13 +802,13 @@ private[org] class BooleanMacro[C <: Context](val context: C, helperName: String
    *   [code generated from callHelper]
    * }
    */
-  def genMacro[T](booleanExpr: Expr[Boolean], methodName: String, clueExpr: Expr[Any]): Expr[T] = {
+  def genMacro[T](booleanExpr: Expr[Boolean], methodName: String, clueExpr: Expr[Any], prettifierExpr: Expr[_], posExpr: Expr[_]): Expr[T] = {
```

Comment:
The reason is that while BooleanMacro sits in scalactic-macro, Prettifier and Position sits in scalactic, which depends on scalactic-macro.  scalactic-macro has no access to Prettifier and Position at that point.